app/enterprise/1.5.x/developer-portal/configuration/index.md

```diff
@@ -5,58 +5,48 @@ title: Dev Portal Configuration
 <div class="docs-grid">
   <h2>Authentication</h2>
```

Comment:
This title doesn't need to be here at all, it only adds confusion.

Reply:
> This title doesn't need to be here at all, it only adds confusion.

Well I think it's probably fine here given the rest of the structure but adding custom fields should move.

![image](https://user-images.githubusercontent.com/3756245/82687486-f617e580-9c1c-11ea-8b2f-d3fd655c7e6a.png)

Reply:
Discussed with @GayleNeumann; holding off on further changes for now due to forthcoming doc site navigation enhancements.